angularjs-portal-home/src/main/webapp/staticFeeds/features.json

```diff
@@ -57,5 +57,23 @@
       "shortDesc" : "If you want to follow NCAA tournaments search for sports in MyUW.",
       "endDate": 1459486800000
     }
+  },
+  {
+    "id": 4,
+    "title": "Wiscard Widget",
+    "description": "MyUW is making your campus life more manageable, one widget at a time. Now you can easily manage your Wiscard account on the go. The new interactive Wiscard widget displays your up-to-date balance and makes it easier for you to add money to your account or report a lost or stolen card. Simply search for \"Wiscard\" and add the interactive widget today!",
+    "img": "img/wiscardWidgets.png",
+    "learnMoreURL": "",
```

Comment:
It's up to you Kim, if you think it would be valuable or not to have.  I don't think the wiscard details page - https://predev.my.wisc.edu/web/apps/details/wiscard-balance?q=wiscard with it's description and screenshot adds a whole lot more to this announcement which already has a description and a screenshot, nor does a deep link to the search results page.  The text already says how to search and add if you want, plus it follows what we've done in the past.

It's up to you if you think we should add something in the 'LearnMoreURL'.  If you look at past examples, it would look like how the 'Welcome to MyUW' learn more link looks.

![image](https://cloud.githubusercontent.com/assets/5521429/14868735/c46d95c8-0c94-11e6-8f26-a00b263f97f4.png)